### ISSUE TITLE
Textsuggest crashing

### ISSUE THREAD

**Issue by nemanjan00** [bharadwaj-raju/TextSuggest#58]
```
 ✘ nemanjan00@nemanjan00-mlaptop  ~  textsuggest --language English
Traceback (most recent call last):
  File "/usr/bin/textsuggest", line 206, in filterSuggestions
    suggestions = self.server.get_suggestions(self.entry.text(), self.languages)
  File "/usr/lib/python3.6/site-packages/dbus/proxies.py", line 145, in __call__
    **keywords)
  File "/usr/lib/python3.6/site-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Python.NameError: Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/dbus/service.py", line 707, in _message_cb
    retval = candidate_method(self, *args, **keywords)
  File "/usr/bin/textsuggest-server", line 276, in get_suggestions
    print('time get_suggestions(%s, %s) =' % (word, language), t2-t1)
NameError: name 'language' is not defined
[1]    29116 abort (core dumped)  textsuggest --language English
```

**Comment by bharadwaj-raju** [comment]
Fixed by commit 0b38a5e4789a.

**Comment by nemanjan00** [comment]
Thanks
